src/Hazelcast.Net.Tests/Networking/AddressProviderTests.cs

```diff
@@ -142,11 +180,15 @@ static void AssertMap(AddressProvider ap, string priv, string pub)
 
                 Assert.That(addressProvider.Map(new NetworkAddress("192.0.0.10")), Is.Null);
 
-                Assert.That(addressProvider.GetAddresses().Count(), Is.EqualTo(4));
-                Assert.That(addressProvider.GetAddresses(), Does.Contain(NetworkAddress.Parse("192.147.0.6:5701")));
-                Assert.That(addressProvider.GetAddresses(), Does.Contain(NetworkAddress.Parse("192.147.0.7:5701")));
-                Assert.That(addressProvider.GetAddresses(), Does.Contain(NetworkAddress.Parse("192.147.0.8:5703")));
-                Assert.That(addressProvider.GetAddresses(), Does.Contain(NetworkAddress.Parse("192.147.0.9:5707")));
+                var (primary, secondary) = addressProvider.GetAddresses();
+                Assert.That(secondary, Is.Empty);
+                Assert.That(primary, Is.EquivalentTo(new[]
+                {
+                    NetworkAddress.Parse("192.147.0.6:5701"),
```

Comment:
Just being sure, in cloud, if no port is supplied within the address, we don't expend it, right?

Reply:
cloud is different, we don't provide addresses / addresses that are provided are ignored / we use the cloud provider which uses an API to retrieve addresses from the cloud service - so yea, no port expansion for cloud - at all.